ui/pages/settings/flask/snaps-list-tab/view-snap/index.scss

```diff
@@ -0,0 +1,84 @@
+div .view-snap-subheader {
+  @media screen and (max-width: $break-small) {
+    display: flex;
+    margin-right: 0;
+    padding: 0 0 16px;
+    flex-direction: column;
+    align-items: center;
+    gap: 8px;
+    height: max-content;
+  }
+}
+
+.view-snap-title {
+  @media screen and (max-width: $break-small) {
+    padding-bottom: 16px;
+  }
+}
+
+.view-snap-pill-toggle-container {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+
+  @media screen and (max-width: $break-small) {
+    width: 100%;
+    justify-content: space-between;
+  }
+}
+
+.view-snap-content-container {
+  @media screen and (max-width: $break-small) {
+    width: 100%;
+  }
+}
+
+.view-snap-pill-container {
+  @media screen and (max-width: $break-small) {
+    padding-left: 0;
+    display: inline-block;
+  }
+}
+
+.view-snap-toggle-container {
+  @media screen and (max-width: $break-small) {
+    padding-left: 0;
+    margin-left: auto;
+    display: inline-block;
+  }
+}
+
+.settings-page__content-item.view-snap-section {
+  border-bottom: 1px solid #d8d8d8;
```

Comment:
Nit: We should really avoid hard-coding colors into stylesheets. This doesn't appear to be one of the design system colors either. Can we use another color here instead from our set of design system colors? (in `ui/css/design-system/colors.scss`)

Reply:
Yeah, no problem -- I'll find an alternative.